extensions/amp-story/1.0/amp-story-store-service.js

```diff
@@ -139,11 +141,13 @@ export const StateProperty = {
 export const Action = {
   ADD_TO_ACTIONS_WHITELIST: 'addToActionsWhitelist',
   CHANGE_PAGE: 'setCurrentPageId',
+  TOGGLE_EMBEDDED_COMPONENT: 'toggleEmbeddedComponent',
```

Comment:
Instead of having two actions, can we have one action that would accept an object parameter `{component: !Element, state: !ComponentState}`?

Reply:
I'll leave this for a follow-up, as I will have to change this anyways for #20186.